install/README.md

```diff
@@ -105,15 +110,15 @@ See [Deploy Management Nodes](deploy_non-compute_nodes.md).
 > **`NOTE`** The PIT node will join Kubernetes after it is rebooted later in
 [Deploy Final NCN](#4-deploy-final-ncn).
 
-## 2. Install CSM Services
+### 2. Install CSM Services
 
 Now that deployment of management nodes is complete with initialized Ceph storage and a running Kubernetes
 cluster on all worker and master nodes, except the PIT node, the CSM services can be installed. The Nexus
 repository will be populated with artifacts; containerized CSM services will be installed; and a few other configuration steps taken.
 
 See [Install CSM Services](install_csm_services.md).
 
-## 3. Validate CSM Health Before Final NCN Deployment
+### 3. Validate CSM Health Before Final NCN Deployment
 
 After installing all of the CSM services, now validate the health of the management nodes and all CSM services.
 The reason to do it now is that if there are any problems detected with the core infrastructure or the nodes, it is
```

Comment:
Two lines after this it talks about redeploying the PIT node, which I thought was language that made you furious!

Reply:
Nice catch, I missed that one.